format/spec.md

```diff
@@ -1134,10 +1148,9 @@ In some cases partition specs are stored using only the field list instead of th
 
 The `field-id` property was added for each partition field in v2. In v1, the reference implementation assigned field ids sequentially in each spec starting at 1,000. See Partition Evolution for more details.
 
-Notes:
+Transforms that accept multiple arguments specify source field IDs using `source-ids` instead of `source-id`. Writers producing these transforms in v1 and v2 metadata should additionally produce the `source-id` field by setting it to the first ID from the `source-ids` list. Writers producing these transforms in v3 metadata should populate only the `source-ids` field because v3 readers will fully-support multi-arg transforms by reading this field.
```

Comment:
Hm, to me , the paragraph's subject is about multi-arg transforms, it is not talking about single arg transform, which should be obvious from the table.  (it may be too confusing if we talk about single-arg transform here as well).

Reply:
How about this?

> In v3 metadata, writers must use only `source-ids` because v3 requires reader support for multi-arg transforms. In v1 and v2 metadata, writers must always write `source-id`; for multi-arg transforms, writers must produce `source-ids` and set `source-id` to the first ID from the field ID list.

That might be more clear.

Reply:
Thanks, it is more clear!